apps/aehttp/test/aehttp_sc_SUITE.erl

```diff
@@ -5741,3 +5756,47 @@ sc_ws_leave_responder_does_not_timeout(Config0) ->
     ct:log("*** Closing ... ***", []),
     ok = sc_ws_close_(Config2).
 
+sc_ws_responder_can_use_any_initiator(Config0) ->
+    Config = sc_ws_open_([{slogan, ?SLOGAN}|Config0],
+                         custom_responder_opts([{initiator_id, any}
+                                               ])),
+    ok = sc_ws_update_(Config),
+    ok = sc_ws_close_(Config).
+
+%% this is an integration test that the enoise session handler is allowing
+%% reconnects later on
+sc_ws_responder_can_use_any_initiator_and_initiator_can_reattach(Config0) ->
+    ct:log("opening channel", []),
+    ct:log("Config0 = ~p", [Config0]),
+    Config = sc_ws_open_([{slogan, ?SLOGAN}|Config0],
+                          custom_responder_opts([ {initiator_id, any}
+                                                , {keep_running, true}
+                                                ])),
+    ct:log("channel opened", []),
+    ct:log("Config = ~p", [Config]),
+    ct:log("*** Leaving channel ***", []),
+
+    Config1 = [{responder_leaves, false}|Config],
+    ct:log("Config1 = ~p", [Config1]),
+    ReestablishOptions = sc_ws_leave_(Config1),
+    Config2 = reconnect_client_(ReestablishOptions, initiator,
+                                [{reconnect_scenario, reestablish} | Config1]),
+    ct:log("*** Verifying that channel is operational ***", []),
+    ok = sc_ws_update_(Config2),
+    ct:log("*** Closing ... ***", []),
+    ok = sc_ws_close_(Config2).
+
+
+custom_responder_opts(L) ->
+    Opts =
+        lists:foldl(
+            fun({keep_running, Bool}, Accum) when is_boolean(Bool) ->
+                Accum#{keep_running => true};
```

Comment:
Might be confusing for future users who might call custom_responder_opts([{keep_running, false}])